release/release.js

```diff
@@ -47,7 +47,7 @@ const devBranchDefault = "dev2"
 const docBranchDefault = "gh-pages2"
 const versionReleasedDefault = VERSION
 const nextVersionDefault = "10.0.0"
-const versionCodeReleasedDefault = 74
+const versionCodeReleasedDefault = 75
```

Comment:
This is just a suggested value. But 75 is the correct code for the next release.